[2025-05-06T15:48:59.015+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-06T15:48:59.054+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: google_maps_reviews.model_data scheduled__2025-05-05T00:00:00+00:00 [queued]>
[2025-05-06T15:48:59.076+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: google_maps_reviews.model_data scheduled__2025-05-05T00:00:00+00:00 [queued]>
[2025-05-06T15:48:59.077+0000] {taskinstance.py:2867} INFO - Starting attempt 4 of 5
[2025-05-06T15:48:59.104+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): model_data> on 2025-05-05 00:00:00+00:00
[2025-05-06T15:48:59.120+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=3212) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-06T15:48:59.125+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'google_maps_reviews', 'model_data', 'scheduled__2025-05-05T00:00:00+00:00', '--job-id', '548', '--raw', '--subdir', 'DAGS_FOLDER/***dag.py', '--cfg-path', '/tmp/tmpua2j20sx']
[2025-05-06T15:48:59.126+0000] {standard_task_runner.py:72} INFO - Started process 3227 to run task
[2025-05-06T15:48:59.127+0000] {standard_task_runner.py:105} INFO - Job 548: Subtask model_data
[2025-05-06T15:48:59.213+0000] {task_command.py:467} INFO - Running <TaskInstance: google_maps_reviews.model_data scheduled__2025-05-05T00:00:00+00:00 [running]> on host 8280c28b7f0f
[2025-05-06T15:48:59.367+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='google_maps_reviews' AIRFLOW_CTX_TASK_ID='model_data' AIRFLOW_CTX_EXECUTION_DATE='2025-05-05T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='4' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-05T00:00:00+00:00'
[2025-05-06T15:48:59.369+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-05-06T15:48:59.370+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-05-06T15:48:59.371+0000] {logging_mixin.py:190} INFO - Current task name:model_data state:running start_date:2025-05-06 15:48:59.056608+00:00
[2025-05-06T15:48:59.372+0000] {logging_mixin.py:190} INFO - Dag name:google_maps_reviews and current dag run status:running
[2025-05-06T15:48:59.372+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-05-06T15:48:59.391+0000] {base.py:84} INFO - Retrieving connection 'postgres_defaut'
[2025-05-06T15:48:59.461+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-05-06T15:48:59.477+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-05-06T15:48:59.478+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=google_maps_reviews, task_id=model_data, run_id=scheduled__2025-05-05T00:00:00+00:00, execution_date=20250505T000000, start_date=20250506T154859, end_date=20250506T154859
[2025-05-06T15:48:59.515+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-05-06T15:48:59.516+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-05-06T15:48:59.517+0000] {logging_mixin.py:190} INFO - Dag name:google_maps_reviews queued_at:2025-05-06 15:44:05.204850+00:00
[2025-05-06T15:48:59.517+0000] {logging_mixin.py:190} INFO - Task hostname:8280c28b7f0f operator:PythonOperator
[2025-05-06T15:48:59.545+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-06T15:48:59.602+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-06T15:48:59.606+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
